src/lib/pages/backoffice/Location/LocationForms/InternalLocationEditor/components/InternalLocationForm/InternalLocationForm.js

```diff
@@ -18,7 +18,7 @@ import { Header, Segment } from 'semantic-ui-react';
 import * as Yup from 'yup';
 
 const InternalLocationSchema = Yup.object().shape({
-  location: Yup.object('missing').shape({
+  location: Yup.object().shape({
```

Comment:
Will this work for newly created internal locations which they don't have a pid yet?

Reply:
Yes, currently if you try to create or edit an internal location in `dev` it stays in loading state for ever. This should fix it. (I thought it was a typo, does that "missing" means something ? )